lib/src/io/output_file_stream.dart

```diff
@@ -1,33 +1,58 @@
-import 'dart:io';
 import 'dart:typed_data';
 
+import 'package:archive/src/io/file_handle.dart';
```

Comment:
Use relative path imports for in-package files

Reply:
Fixed